test/common/upstream/utility.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "common/common/utility.h"
+#include "common/config/cds_json.h"
+#include "common/json/json_loader.h"
+
+namespace Envoy {
+namespace Upstream {
+namespace {
+
+inline std::string defaultSdsClusterJson(const std::string& name) {
+  return fmt::sprintf(R"EOF(
+  {
+    "name": "%s",
+    "connect_timeout_ms": 250,
+    "type": "sds",
+    "lb_type": "round_robin"
+  }
+  )EOF",
+                      name);
```

Comment:
Don't see an option for this at https://clang.llvm.org/docs/ClangFormatStyleOptions.html.